aydin/nn/models/jinet.py

```diff
@@ -106,6 +106,9 @@ def __init__(
         self.compile(optimizer=Adam(learning_rate=learning_rate), loss='mse')
         self.compiled = True
 
+    def need_batch_size_one(self):
```

Comment:
What is this for? It seems to return different types of output than that in unet.py.